### ISSUE TITLE
Focusing

### ISSUE THREAD

**Issue by ispybadguys** [coral/freemote#2]
Is it possible to control the focus in manual focus mode by stepping in or out? I have seen some information on this at https://gregleeds.com/reverse-engineering-sony-camera-bluetooth/. I tried using the Nordic App on Android.

Thanks

Kurt

**Comment by coral** [comment]
@ispybadguys probably? never tried it. feel free to run the code and try

**Comment by ClaesClaes** [comment]
I have not tried but it's possible. There's an iPhone app called 'sMote - for Sony cameras' on App Store doing exactly this. Adjusting focus via BLE.

**Comment by ispybadguys** [comment]
Maybe you know something that I don't. (this is almost certain). I discovered a small error in Greg's commands. The Focus Near consists of a Focus Near Button Down command 0x02 0x6b 0x20 followed by a Focus Near Button Up command 0x02 0x6a 0x00. These commands are 3 bytes long. I tested the commands using Nordic's Android App and it works if I send these as a byte array but the Adafruit library doesn't support this. I tried writing it using 6 each .write8_resp() but the camera doesn't respond. I also tried 6 each .write8() with no joy. Any ideas?

Thanks

Kurt
